project/AppDependencies.scala

```diff
@@ -6,13 +6,12 @@ object AppDependencies {
 
   val compile = Seq(
     ws,
-    "uk.gov.hmrc"         %% "play-breadcrumb"          % "1.0.0",
     "uk.gov.hmrc"         %% "bootstrap-play-26"        % "1.5.0",
     "uk.gov.hmrc"         %% "play-partials"            % "6.9.0-play-26",
     "uk.gov.hmrc"         %% "url-builder"              % "3.3.0-play-26",
     "uk.gov.hmrc"         %% "http-caching-client"      % "9.0.0-play-26",
     "uk.gov.hmrc"         %% "play-language"            % "4.2.0-play-26",
-    "uk.gov.hmrc"         %% "local-template-renderer"  % "2.8.0-play-26",
+    "uk.gov.hmrc"         %% "local-template-renderer"  % "2.8.1-play-26",
```

Comment:
The latest version is 2.9.0-play-26. Please update the same.

Reply:
Done